tests/Makefile.tests_common

```diff
@@ -4,17 +4,25 @@ APPLICATION ?= tests_$(notdir $(patsubst %/,%,$(CURDIR)))
 TESTBASE ?= $(CURDIR)/../..
 RIOTBASE ?= $(TESTBASE)/RIOT
 BUILD_DIR ?= $(TESTBASE)/build
+
+-include $(TESTBASE)/dist/etc/conf/$(BOARD).env
+
 # Default timeout for serial connection to a board
-CMD_TIMEOUT ?= 1
+HIL_CMD_TIMEOUT ?= 1
+export HIL_CMD_TIMEOUT
+CMD_TIMEOUT ?= HIL_CMD_TIMEOUT
 export CMD_TIMEOUT
 # Default wait time before connecting the serial
-CONNECT_WAIT ?= 0
+HIL_CONNECT_WAIT ?= 0
+export HIL_CONNECT_WAIT
+CONNECT_WAIT ?= HIL_CONNECT_WAIT
 export CONNECT_WAIT
-# default periph/timer config
-PERIPH_TIMER_DEV ?= 0
-export PERIPH_TIMER_DEV
-PERIPH_TIMER_HZ ?= 1000000
-export PERIPH_TIMER_HZ
+
+HIL_RESET_WAIT ?= 3
+export HIL_RESET_WAIT
+RESET_WAIT ?= HIL_RESET_WAIT
```

Comment:
please remove these duplicate (old) variables. Looks like a work around to me, we should rather adapt the tests to use the new env variable names.

Reply:
OK, thought we would do a step by step but I can adapt everything.